### PR TITLE
openssh: support spack external find

### PR DESCRIPTION
**Pull request by sethrj**

### CODE REVIEW

**Review by alalazo — CHANGES_REQUESTED** [review]
LGTM, but not to `flake8`:
```
var/spack/repos/builtin/packages/openssh/package.py:55: [E303] too many blank lines (2)
```

File: var/spack/repos/builtin/packages/openssh/package.py [hunk 1]
```diff
@@ -40,6 +41,17 @@ class Openssh(AutotoolsPackage):
...
+    # command 'ssh'
+    executables = ['^ssh$', '^rsh$']
+
+    @classmethod
+    def determine_version(cls, exe):
+        output = Executable(exe)('-V', output=str, error=str).rstrip()
+        match = re.search(r'OpenSSH_([^, ]+)', output)
+        return match.group(1) if match else None
+
+
```
Comment (haampie) [inline var/spack/repos/builtin/packages/openssh/package.py]:
```suggestion
```

**Review by haampie — APPROVED** [review]

**Review by alalazo — APPROVED** [review]

**Comment by sethrj** [comment]
Thanks @haampie @alalazo -- my bad for not checking flake8 locally.

**Comment by haampie** [comment]
seems like gitlab-ci is stuck

**Pull request merged**